recipe/build.sh

```diff
@@ -102,9 +102,11 @@ cmake .. -LAH                                                             \
     -DWITH_FFMPEG=1                                                       \
     -DWITH_MATLAB=0                                                       \
     -DWITH_VTK=0                                                          \
+    -DWITH_QT=5                                                           \
     -DWITH_GPHOTO2=0                                                      \
     -DINSTALL_C_EXAMPLES=0                                                \
     -DOPENCV_EXTRA_MODULES_PATH="../opencv_contrib-$PKG_VERSION/modules"  \
+    -DBUILD_opencv_cvv=OFF                                                \
```

Comment:
I guess it just depends if anyone is using it? We mostly need to test this highgui stuff on OSX as I believe OSX is currently working.